theme-release-110.rst

```diff
@@ -0,0 +1,81 @@
+.. post:: October 18, 2022
```

Comment:
```suggestion
.. post:: Nov 1, 2022
```

We've just announced the 1.1.0b3 on twitter. Let's wait a week and then plan to ship it on Nov 1?